src/gui/shader.cpp

```diff
@@ -1,26 +1,33 @@
 #include "shader.hpp"
 
+#include "../logging.hpp"
+
 #include <GL/glew.h>
 
 #include <filesystem>
 #include <fstream>
-#include <iostream>
 #include <string>
 #include <vector>
 
+// TODO(alem): what do we do on error
 GLuint
 load_shaders(
     const std::filesystem::path& vertex_file, const std::filesystem::path& fragment_file
-) {
+)
+{
```

Comment:
This format is wrong

Reply:
No

Reply:
please elaborate. I though curly brackets went on the same line as the closing parentheses.

Reply:
Not on function defs